tests/src/test/java/alluxio/master/block/BlockMasterIntegrityIntegrationTest.java

```diff
@@ -54,4 +63,29 @@ public void deleteOrphanedBlocks() throws Exception {
         (v) -> newWorker.getStoreMetaFull().getNumberOfBlocks() == 0,
         WaitForOptions.defaults().setTimeoutMs(2000));
   }
+
+  @Test
+  public void deleteInvalidBlocks() throws Exception {
+    AlluxioURI uri = new AlluxioURI("/test");
+    int len = 10;
+    FileSystem fs = mCluster.getClient();
+    BlockWorker worker = mCluster.getWorkerProcess().getWorker(BlockWorker.class);
+    FileSystemTestUtils.createByteFile(fs, uri, WriteType.MUST_CACHE, len);
+    Assert.assertEquals(1, worker.getStoreMetaFull().getNumberOfBlocks());
+    FileSystemMaster fsm =
+        mCluster.getLocalAlluxioMaster().getMasterProcess().getMaster(FileSystemMaster.class);
+    InodeTree tree = Whitebox.getInternalState(fsm, "mInodeTree");
+    LockedInodePath path = tree.lockInodePath(uri, InodeTree.LockMode.WRITE);
+    DeleteOptions options = DeleteOptions.defaults();
+    JournalContext ctx = Whitebox.invokeMethod(fsm, "createJournalContext");
```

Comment:
instead of using whitebox, do you think it makes sense to directly call `removeBlocks` on the `BlockMaster`? The `DefaultBlockMaster` instance is available via `mCluster.getLocalAlluxioMaster().getMasterProcess().getMaster(BlockMaster.class)`